packages/contracts-bedrock/src/cannon/libraries/MIPSSyscalls.sol

```diff
@@ -53,6 +53,7 @@ library MIPSSyscalls {
     uint32 internal constant SYS_PRLIMIT64 = 4338;
     uint32 internal constant SYS_CLOSE = 4006;
     uint32 internal constant SYS_PREAD64 = 4200;
+    uint32 internal constant SYS_STAT = 4106;
```

Comment:
This doesn't require a version bump for MIPS.sol because this constant isn't used there?  Is that right?

Reply:
Correct

Reply:
Indeed. Since it's unused the evm source code is unaltered.